controllers/grafana_controller.go

```diff
@@ -213,20 +213,14 @@ func (r *GrafanaReconciler) syncStatuses(ctx context.Context) error {
 		return err
 	}
 
-	contactPoints := &grafanav1beta1.GrafanaContactPointList{}
-	err = r.List(ctx, contactPoints)
-	if err != nil {
-		return err
-	}
-
-	notificationTemplates := &grafanav1beta1.GrafanaNotificationTemplateList{}
-	err = r.List(ctx, notificationTemplates)
+	folders := &grafanav1beta1.GrafanaFolderList{}
+	err = r.List(ctx, folders)
 	if err != nil {
 		return err
 	}
 
-	muteTimings := &grafanav1beta1.GrafanaMuteTimingList{}
-	err = r.List(ctx, muteTimings)
+	libraryPanels := &grafanav1beta1.GrafanaLibraryPanelList{}
```

Comment:
I see that muteTimings and notificationTemplates were removed, is that for a reason?